extern/boostd-data/ldb/service.go

```diff
@@ -50,13 +50,11 @@ func (s *Store) Start(_ context.Context) error {
 		return err
 	}
 
-	if s.repopath == "" {
-		// used by tests - prepare db with a cursor
-		log.Debug("preparing db with next cursor")
-		s.db.SetNextCursor(context.Background(), 100)
-	}
+	// Prepare db with a cursor
+	log.Debug("preparing db with next cursor")
+	s.db.SetNextCursor(ctx, 100)
```

Comment:
This should only be done once when we `init` the database, not on every `Start`.